examples/toolchain_v1alpha1_usersignup_cr.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: toolchain.dev.openshift.com/v1alpha1
-kind: UserProvisionRequest
+kind: UserSignup
```

Comment:
This CR example has `targetCluster` set but `approved` is missing.  We should add `approved: true` .. right ??